src/components/PieChartFacetContents.js

```diff
@@ -17,10 +17,21 @@ type PieChartFacetContentsProps = {
    * the map are ignored.
    */
   entityColors: Map<string, string>;
+  /** Property to align the legend to left, right or center. */
```

Comment:
The comments for properties that have default values should indicate what the default values are. So the user knows what will happen if they omit the property.

Reply:
Should I include the option values as well in the comments apart from default values? For e.g "Property representing horizontal alignment of the legend box within the chart area. Possible values are left, right or center. Default value is center." Or I should omit the Possible values part and just keep the default values since we are mentioning in the actual property declaration what the possible values are...

Reply:
Yes, that sounds good with the options included.